### PR TITLE
Static Postcss 2.1.1

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

**Comment by adamweeks** [comment]
https://github.com/postcss/postcss-loader/issues/347

File: package.json [hunk 1]
```diff
@@ -157,10 +157,10 @@
     "mkdirp": "^0.5.1",
     "mockdate": "^2.0.1",
     "npm-check-updates": "^2.12.1",
-    "postcss": "^6.0.3",
+    "postcss": "^6.0.20",
```
Comment (bzang) [inline package.json]: Should we also set this as an exact version like `postcss-load`?

**Comment by adamweeks** [comment]
Deploy preview for *hopeful-hamilton-4090b9* ready!

Built with commit 3e52471ac5b0d6a08ee4c16dd8b5caa6f33d2bac

https://deploy-preview-696--hopeful-hamilton-4090b9.netlify.com

File: package.json (hunk 1 above)
Comment (adamweeks) [reply package.json]: The loader is just the issue, we could probably remove this bump if we wanted to

**Pull request merged**